.circleci/config.yml

```diff
@@ -107,14 +107,16 @@ commands:
   perform-build:
     description: "Builds Unity project"
     parameters:
+      project:
+        type: string
       target:
         type: string
     steps:
       - run:
           name: Build Unity project and export it
           command: |
             /opt/unity/Editor/Unity \
-            -projectPath IntegrationTests \
+            -projectPath << parameters.project >> \
```

Comment:
this is definitely easier to work with. 

But I do have one concern, based on a lot of painful experience on iOS: 

If we don't have _any_ test projects that use the old integration method, it's almost guaranteed that we're going to break it. 

That's why we ended up with [these multiple projects](https://github.com/RevenueCat/purchases-ios/tree/main/Tests/InstallationTests) on iOS: we kept making updates to something in Cocoapods and Carthage would break, or swift package manager, or whatever. 

So I'd say we need to either: 
- keep one of these using the old `unityPackage`
- have a new one that's a basic copy, using the old `unityPackage`

Either way we just need to know if we break the `unityPackage`

Reply:
maybe that can be a separate PR or something, but I would try to not consider the Epic "done" until we take care of it, because it might be a while before we come back to this and we might have forgotten how it works by then

Reply:
@aboedo We have 2 projects. `Subtester` using purchases-unity through UPM (as a local package) and `IntegrationTests` using a `unitypackage`. Both projects are built and exported to iOS and Android including the APITests so we compile APITests for both projects to ensure the API is the same on both.

That said, `IntegrationTests` needs to have a `unitypackage` first, that's why we use `Subtester` to generate the `unitypackage` first, then import it to `IntegrationTests` and then build and export `IntegrationTests`. 

So basically we don't have a project using a `unitypackage` where we can test manually, only in automated tests. We could have another project where we use the `unitypackage` system manually but I'm worried of overcomplicating the repo and I don't think it's that valuable. Do lmk if you were thinking of something else!

Reply:
not having manual tests is fine, I just want to make sure at least one status check fails if we fail to deliver a working `unityPackage`